app/controllers/twitch_stream_updates_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TwitchStreamUpdatesController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def show
+    if params["hub.mode"] == "denied"
+      Rails.logger.error("Twitch Webhook was denied: #{params.to_json}")
+      head :no_content
+    else
+      render plain: params["hub.challenge"]
+    end
+  end
+
+  def create
+    head :no_content
+
+    unless secret_verified?
+      Rails.logger.warn("Twitch Webhook Recieved for which the webhook could not be verified")
```

Comment:
This also might warrant an error tracker or something. This _might_ mean that someone is attempting to spoof the Twitch webhooks. OR there is an issue with the integration